andela/ah-baqi-frontend
# #166980204 Setup Sass Processing

### What does this PR do?
Adds a Sass preprocessor
### How should this be manually tested?
- Checkout to ch-setup-sass-166980204
- Install dependencies: yarn
- Run the local server: yarn develop
- Navigate to http://localhost:3000
- The text should be blue
### Any background context you want to provide?
None
### What are the relevant pivotal tracker stories?
[#166980204](https://www.pivotaltracker.com/story/show/166980204)
### Screenshots (if appropriate)
